docs/dev/developer-guides/building-on-zksync/events.md

```diff
@@ -0,0 +1,40 @@
+# zkSync events
+
+## Overview 
+Given that smart contracts themselves cannot listen to any events, events are a mechanism to publish actions to listeners outside the blockchain.
+
+The blockchain makes all information available to the public, and any actions can be discovered by carefully looking into the transactions, but events are a shortcut for making the development of external systems work with smart contracts; they let Dapps keep track of and respond to what's happening to a smart contract. They can also be searched for because they are indexable. Therefore, you should emit an event anytime something occurs in your smart contract that some system outside the blockchain should be aware of so that the outside system may listen for such occurrences. 
```

Comment:
Blockchains are public by design, and therefore makes all information available to the public, and any actions can be discovered by carefully looking into the transactions. Events are a shortcut for making specific information easily available for external systems; they let dApps keep track of, and respond to what's happening to a smart contract.